parts/windows/containerdtemplate.toml

```diff
@@ -14,7 +14,7 @@ state = "C:\\ProgramData\\containerd\\state"
   level = "info"
 
 [metrics]
-  address = ""
+  address = "0.0.0.0:10257"
```

Comment:
Q: Who will consume this, metric server and omsagent?

Reply:
So far, we use prometheus service discovery to find Kubernetes nodes and replace the metric URL with a proxied one.

Reply:
@keikhara, Do we have any concerns from the point of security, as we expose 0.0.0.0 and 10257?  Linux is using this endpoint for containerd since June.